src/Conference/ConferenceContainer.jsx

```diff
@@ -47,6 +47,8 @@ const ConferenceContainer = () => {
           id={id}
           setConference={setConference}
           setUpdatedConference={setUpdatedConference}
+          isLoading={isLoading}
+          errorCaught={errorCaught}
```

Comment:
Nit: It's probably fine just to name this error, errorCaught seems unnecessary ;)